[2022-03-17 19:47:02,304] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-03-17 19:47:02,315] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-03-17 19:47:02,315] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 19:47:02,315] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-17 19:47:02,315] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 19:47:02,326] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): remove_files_task> on 2018-01-01 00:00:00+00:00
[2022-03-17 19:47:02,332] {standard_task_runner.py:52} INFO - Started process 567 to run task
[2022-03-17 19:47:02,336] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_project', 'remove_files_task', 'scheduled__2018-01-01T00:00:00+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_project.py', '--cfg-path', '/tmp/tmp1p40ycwd', '--error-file', '/tmp/tmpvxwjtytn']
[2022-03-17 19:47:02,337] {standard_task_runner.py:77} INFO - Job 19: Subtask remove_files_task
[2022-03-17 19:47:02,401] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2018-01-01T00:00:00+00:00 [running]> on host 06aea6ade097
[2022-03-17 19:47:02,441] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-17 19:47:02,462] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_project
AIRFLOW_CTX_TASK_ID=remove_files_task
AIRFLOW_CTX_EXECUTION_DATE=2018-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2018-01-01T00:00:00+00:00
[2022-03-17 19:47:02,463] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 19:47:02,464] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/dft-road-casualty-statistics-accident-2018.csv /opt/***/dft-road-casualty-statistics-accident-2018.parquet']
[2022-03-17 19:47:02,476] {subprocess.py:85} INFO - Output:
[2022-03-17 19:47:02,485] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-17 19:47:02,510] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_project, task_id=remove_files_task, execution_date=20180101T000000, start_date=20220317T194702, end_date=20220317T194702
[2022-03-17 19:47:02,548] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-17 19:47:02,578] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-20 06:17:43,578] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-03-20 06:17:43,589] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2018-01-01T00:00:00+00:00 [queued]>
[2022-03-20 06:17:43,590] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 06:17:43,590] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-20 06:17:43,590] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 06:17:43,601] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): remove_files_task> on 2018-01-01 00:00:00+00:00
[2022-03-20 06:17:43,607] {standard_task_runner.py:52} INFO - Started process 341 to run task
[2022-03-20 06:17:43,612] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion_gcs_dag_project', 'remove_files_task', 'scheduled__2018-01-01T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag_project.py', '--cfg-path', '/tmp/tmp8gw7fcy1', '--error-file', '/tmp/tmphe9oxz8x']
[2022-03-20 06:17:43,613] {standard_task_runner.py:77} INFO - Job 40: Subtask remove_files_task
[2022-03-20 06:17:43,678] {logging_mixin.py:109} INFO - Running <TaskInstance: data_ingestion_gcs_dag_project.remove_files_task scheduled__2018-01-01T00:00:00+00:00 [running]> on host 974079c1cd0d
[2022-03-20 06:17:43,722] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-03-20 06:17:43,744] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_ingestion_gcs_dag_project
AIRFLOW_CTX_TASK_ID=remove_files_task
AIRFLOW_CTX_EXECUTION_DATE=2018-01-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2018-01-01T00:00:00+00:00
[2022-03-20 06:17:43,745] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-20 06:17:43,746] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'rm /opt/***/dft-road-casualty-statistics-accident-2018.csv']
[2022-03-20 06:17:43,759] {subprocess.py:85} INFO - Output:
[2022-03-20 06:17:43,771] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-20 06:17:43,800] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=data_ingestion_gcs_dag_project, task_id=remove_files_task, execution_date=20180101T000000, start_date=20220320T061743, end_date=20220320T061743
[2022-03-20 06:17:43,824] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-20 06:17:43,856] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
